README.md

```diff
@@ -20,6 +20,15 @@ This section describes how to install the plugin and get it working.
 
 ## Releases
 
+### 1.4.0
+https://github.com/Cinode-Labs/cinode-recruitment/releases/tag/v1.4
+
+Update with the following feature:
+* Option to protect from spam with Google Recaptcha
+* Optional field set Available from date
+* Optional field for select Pipeline, Multiple pipelines
```

Comment:
What is "multiple pipelines" feature? Is it the ability to put candidates i certain pipe/stage depending on conditionals? Maybe slighly expand text, unless widely known what multiple pipelines means.

Reply:
Candidate can select option from dropdown list of available Pipelines